docs/content/en/docs/packages/overview.md

```diff
@@ -12,7 +12,14 @@ weight: 1
 Components of EKS Anywhere curated packages consist of a controller, a CLI, and artifacts.
 
 ### Package controller
-The package controller will install, upgrade, configure and remove packages from the cluster. The package controller will watch the packages and packagebundle custom resources for the packages to run and their configuration values. The package controller only runs on the management cluster and manages packages on the management cluster and on the workload clusters.
+The package controller will install, upgrade, configure and remove packages from the cluster.
+
+The package controller performs these actions by watching the package and packagebundle custom resources to determine which packages to run and their appropriate configuration values.
+
+Packages CRD resources map to helm charts that the package controller uses to install packages workloads (such as cluster-autoscaler or metrics-server) on your clusters. The packagebundle object is the mapping between the package name and the specific helm chart and images that will be installed.
+
+The package controller only runs on the management cluster and manages packages on the management cluster and on the workload clusters.
+To learn more about how to install packages on both management and workload clusters, see below.
 
 Package release information is stored in a package bundle manifest. The package controller will continually monitor and download new package bundles. When a new package bundle is downloaded, it will show up as update available and users can use the CLI to activate the bundle to upgrade the installed packages.
```

Comment:
> Package release information is stored in a package bundle manifest

Starting to get a little confused here. There are packages custom resources, packagebundle custom resource, helm charts, and now a package bundle manifest. I think we need to more clearly explain what each one of these things is, and how they relate to one another.

Reply:
Ah looks like we may explain this in more detail below. We may want to consider moving the order around here, or linking to the below in this section.